src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Staking withdrawals
+description: Page summarizing what staking push withdrawals are, how they work, and what stakers need to do to get their rewards
+lang: en
+template: staking
+image: ../../../assets/staking/leslie-withdrawal.png
+alt: Leslie the rhino with her staking rewards
+sidebarDepth: 2
+summaryPoints:
+  - The Shanghai upgrade brings staking withdrawals to Ethereum
+  - Validator operators must provide a withdrawal address to enable
+  - Rewards payments are automatically sent to the address provided every few days
+  - Fully exited validators will automatically receive their remaining balance
+---
+
+<UpgradeStatus dateKey="page-upgrades-withdrawals">
+  Staking withdrawals will be enabled through the Shanghai/Capella upgrade. This upgrade is expected to take place in the first half of 2023. <a href="#when">More below</a>
+</UpgradeStatus>
+
+The Shanghai/Capella upgrade enables **staking withdrawals** on Ethereum, allowing people to unlock rewards and optionally fully withdrawal funds from staking. Rewards payments will automatically and regularly be sent to a provided withdrawal address linked to each validator. Users can also exit staking entirely, unlocking their full validator balance.
+
+**Note to validator operators**: If you did not provide a withdrawal address on initial deposit, you will need to upgrade withdrawal credentials before any withdrawals will be enabled for your validator account. The [Staking Launchpad](https://launchpad.ethereum.org/withdrawals) has more details on when and how to do this.
+
+## Staking rewards {#staking-rewards}
+
+**Rewards payments** are automatically processed for active validator accounts with a maxed out effective balance of 32 ETH, who are accumulating rewards.
+
+Any balance above 32 ETH earned through rewards does not actually contribute to principle, or increase the sway of this validator on the network, and is thus automatically withdrawn as a rewards payment every few days. Aside from providing a withdrawal address one time, these rewards do not require any action from the validator operator, nor do they consume gas (no associated transaction fee) or take up existing block space.
```

Comment:
You mention this does not consume gas but I think perhaps worth explicitly mentioning this whole process is accomplished by broadcasting messages, NOT on-chain txs?

Reply:
Ah, I think I see your point here.. 

> Aside from providing a withdrawal address one time, these rewards do not require any action from the validator operator, nor do they consume gas (no associated transaction fee) or take up existing block space.

...Kind of implied that "providing a withdrawal address one time" DOES cost gas and take up block space.

I can touch this up.